docs/source/faq.rst

```diff
@@ -97,3 +97,32 @@ For instance, you can stop showing each trial result as follows:
 
 
 Please refer to :class:`optuna.logging` for further details.
+
+
+How to save a machine learning model trained in an objective function?
+----------------------------------------------------------------------
+
+Optuna saves hyperparameter values with its corresponding objective value to storage,
+but it discards intermediate objects such as machine learning models and neural network weights.
+To save models or weights, please use features of the machine learning library you used.
+
+We recommend saving :obj:`~optuna.trial.Trial.trail_id` with a model in order to identify its corresponding trial.
+For example, you can save SVM models trained in the objective function as follows:
+
+.. code-block:: python
+
+    def objective(trial):
+        svc_c = trial.suggest_loguniform('svc_c', 1e-10, 1e10)
+        clf = sklearn.svm.SVC(C=svc_c)
+        clf.fit(X_train, y_train)
+
+        # Save a trained model to a file.
+        joblib.dump(clf, '{}.joblib'.format(trial.trial_id))
```

Comment:
How about using `pickle` instead of `joblib` here? I understand that the official sklearn document (https://scikit-learn.org/stable/modules/model_persistence.html) says `joblib` is more preferable than `pickle`, but sklearn also works with `pickle`. I think non-negligible amount of users do not know `joblib`, and users might be more familiar with `pickle`.

Reply:
That sounds good. As you mentioned, `pickle` is much more familiar to Python users than `joblib`. I'll fix it.